### ISSUE TITLE
Stretched unread count in iOS client

### ISSUE THREAD

**Issue by rabryst** [Ranchero-Software/NetNewsWire#4029]
In iOS client version 6.1.2 (build 6116), a weird glitch shows a stretched image for the unread count. This goes away when restarting the app after force-quitting.
![IMG_2078](https://github.com/Ranchero-Software/NetNewsWire/assets/3165117/3d3d2480-7e5a-4563-b1dc-c60d7b0a8e10)

**Comment by brentsimmons** [comment]
Still seeing this?

**Comment by rabryst** [comment]
Nope! I forgot I even reported it. I guess the iOS update *du jour* took care of it.
